2009-11-22  Hubert Chathi  <uhoreg@debian.org>
	    Yavor Doganov  <yavor@gnu.org>

	* Source/art/FTFontEnumerator.m (load_font_configuration):
          Add /var/lib/GNUstep to the search path; Debian-specific.

--- gnustep-back.orig/Source/art/FTFontEnumerator.m
+++ gnustep-back/Source/art/FTFontEnumerator.m
@@ -350,7 +350,11 @@
   families_seen = [[NSMutableSet alloc] init];
   families_pending = [[NSMutableSet alloc] init];
 
-  paths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSAllDomainsMask, YES);
+  /* Search for fonts in all domains including those generated by
+     mknfonts in postinst.  */
+  paths = [NSSearchPathForDirectoriesInDomains (NSLibraryDirectory,
+						NSAllDomainsMask, YES)
+	      arrayByAddingObject: @"/var/lib/GNUstep"];
   for (i = 0; i < [paths count]; i++)
     {
       path = [paths objectAtIndex: i];
